predicators/datasets/vlm_input_data.py

```diff
@@ -0,0 +1,665 @@
+"""Functions to create offline demonstration data by leveraging VLMs."""
+
+import ast
+import glob
+import logging
+import os
+import re
+from pathlib import Path
+from typing import Dict, List, Sequence, Set, Tuple
+
+import numpy as np
+import PIL.Image
+
+from predicators import utils
+from predicators.envs import BaseEnv
+from predicators.envs.vlm_envs import VLMPredicateEnv
+from predicators.pretrained_model_interface import GoogleGeminiVLM, \
+    VisionLanguageModel
+from predicators.settings import CFG
+from predicators.structs import Action, Dataset, GroundAtom, \
+    ImageOptionTrajectory, LowLevelTrajectory, Object, ParameterizedOption, \
+    Predicate, State, Task, _Option
+
+
+def _generate_prompt_for_atom_proposals(
+        traj: ImageOptionTrajectory, trajectory_subsample_freq: int
+) -> List[Tuple[str, List[PIL.Image.Image]]]:
+    """Prompt for generating proposals for atoms."""
+    ret_list = []
+    if CFG.grammar_search_vlm_atom_proposal_prompt_type == "naive_each_step":
+        prompt = (
```

Comment:
We're going to keep iterating on these prompts, and we'll probably want to easily copy and paste them from here into the VLM web interface for quickly debugging/testing outputs from the VLM. But copy and pasting this is annoying since we have to remove the quotes and stuff. Thoughts on moving the prompts to a separate file (either .txt or .py) where they are one string in one line? We could also have them as one line in this file, but I think we're probably going to have many more prompt options we want to pick from so a dedicated file for it might be ok.